elimar/modules/users/urls.py

```diff
@@ -1,11 +1,16 @@
 from django.urls import path
 
-from .views import FirstUserSignUpView
+from .views import FirstUserSignUpView, UsersList
 
 urlpatterns = [
     path(
         "first-signup/",
         FirstUserSignUpView.as_view(),
         name="first-sign-up",
     ),
+    path(
+        "list/",
```

Comment:
I would prefer just "", so the URL will look like "/users/"
```suggestion
        "",
```
Wdyt?

Reply:
Ahh sorry, it was late when I did this 😨